index.js

```diff
@@ -242,9 +249,9 @@ TCurl.prototype.request = function tcurlRequest(opts, delegate) {
         }
     });
 
-    client.waitForIdentified({
-        host: opts.hostname + ':' + opts.port
-    }, onIdentified);
+    var peer = subChan.peers.choosePeer();
```

Comment:
health checks may not want to use the -H option. It should always go to a specific worker. However, the point about evenly distribute tcurl requests is valid.